railties/lib/rails/application/configuration.rb

```diff
@@ -33,7 +33,7 @@ def initialize(*)
         @filter_parameters                       = []
         @filter_redirect                         = []
         @helpers_paths                           = []
-        @hosts                                   = Array(([IPAddr.new("0.0.0.0/0"), IPAddr.new("::/0"), ".localhost"] if Rails.env.development?))
+        @hosts                                   = Array((["localhost", IPAddr.new("0.0.0.0/0"), IPAddr.new("::/0"), ".localhost"] if Rails.env.development?))
```

Comment:
Instead of adding `localhost` can we move the `.localhost` to the front?

Reply:
Yup